administrator/language/en-GB/en-GB.com_joomgallery.ini

```diff
@@ -1671,4 +1651,46 @@ COM_JOOMGALLERY_MAIMAN_NT_ALERT_RESET_NAMETAGS_CONFIRM="Are you sure you want to
 COM_JOOMGALLERY_MAIMAN_ALERT_RESET_VOTES_CONFIRM="Are you sure you want to proceed? Some or all votes or all vote IP addresses might be deleted!"
 COM_JOOMGALLERY_CONFIG_GS_TAB_BACKEND_REPLACEMENTS="Special characters"
 COM_JOOMGALLERY_CONFIG_UR_UU_UPLOAD_BATCH_LONG="Enable this option if the frontend users should be able to use the Batch Upload.<br />Note that there is no security check for zip files. You should only enable this option if it is ensured that the zip files contain only secure image files!"
-COM_JOOMGALLERY_UPLOAD_BATCH_UPLOAD_NOTE="Please do not upload zipped directories!<br />Only zipped image files will be stored correctly.<br />Note that there is no security check for zip files. Only upload zip files if you are sure that the zip files contain only secure image files!"
\ No newline at end of file
+COM_JOOMGALLERY_UPLOAD_BATCH_UPLOAD_NOTE="Please do not upload zipped directories!<br />Only zipped image files will be stored correctly.<br />Note that there is no security check for zip files. Only upload zip files if you are sure that the zip files contain only secure image files!"
+;-------
+;Changed/Added with the PR "JoomIMGtools"
+;------- 
+COM_JOOMGALLERY_UPLOAD_CREATE_THUMBNAIL_FROM="Resize image from"
+COM_JOOMGALLERY_UPLOAD_RESIZE_TO_MAX="Resizing to max edge lenght using "
```

Comment:
Check spelling: leng**th**

Reply:
Thanks!
Seems that the constant COM_JOOMGALLERY_UPLOAD_CREATE_THUMBNAIL_FROM first has been changed by you but now is not used anymore. You should move it with it' original value to section of the removed constants.